aspnetcore/signalr/hubcontext.md

```diff
@@ -0,0 +1,62 @@
+---
+title: SignalR HubContext
+author: rachelappel
+description: Learn how to use the ASP.NET Core SignalR HubContext service for sending notifications to clients from outside a hub.
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 06/13/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/HubContext
+---
+# Send messages from outside a hub
+
+By [Mikael Mengistu](https://github.com/mikaelm12)
```

Comment:
super nit: you can do your twitter handle

Reply:
Oh, nice